### PR TITLE
BGZIP and MT writer

### PR DESCRIPTION
**Pull request by hopedisastro**
Script 1 bgzips a VCF file, writing to output and a separate script reads the bgzip file into a hail matrix table. Outputs Hail matrix table to a GCP bucket, ready for downstream analysis with Hail query.

### CODE REVIEW

**Comment by hopedisastro** [comment]
I couldn't get it to work as one script. But I think it's ok to keep it separate like this - allows for greater flexibility in the future

File: str/helper/bgzip.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            'vcf.bgz.tbi': '{root}.vcf.bgz.tbi',
+        }
+    )
+
+    bcftools_job.command(
+        f"""
+
+        bcftools view {vcf_input} | bgzip -c >  {bcftools_job.vcf_output['vcf.bgz']}
+
+        tabix -f -p vcf {bcftools_job.vcf_output['vcf.bgz']}
```
Comment (MattWellie) [inline str/helper/bgzip.py:56]: -f shouldn't be required here, this is a freshly created name

**Review by MattWellie — APPROVED** [review]
LGTM

File: str/helper/bgzip.py (hunk 1 above)
Comment (hopedisastro) [inline str/helper/bgzip.py:56]:
```suggestion
```

Comment (hopedisastro) [reply str/helper/bgzip.py:56]: remove tabix

File: str/helper/bgzip.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    bcftools_job = b.new_job(name=f'{input_file_name} BGZIPPING')
+    bcftools_job.image(BCFTOOLS_IMAGE)
+    bcftools_job.memory(job_memory)
+    bcftools_job.storage(job_storage)
+    bcftools_job.cpu(job_cpu)
+
+    bcftools_job.declare_resource_group(
+        vcf_output={
+            'vcf.bgz': '{root}.vcf.bgz',
+            'vcf.bgz.tbi': '{root}.vcf.bgz.tbi',
```
Comment (hopedisastro) [inline str/helper/bgzip.py:47]:
```suggestion
```

**Pull request merged**